DOItools.php

```diff
@@ -568,7 +578,7 @@ function checkTextForMetas($text){
     $authors = formatAuthors($author_list, true);
     foreach ($authors as $no => $auth) {
       $names = explode (', ', $auth);
-      $newp["author" . ($no + 1)] = $names[0];
```

Comment:
This strikes me as something I might have done intentionally - perhaps I check elsewhere if the author parameter is checked, but not the last parameter.  Have you worked this through for unintended consequences?